plugin/sdk/base.py

```
@@ -0,0 +1,140 @@
+    def __init__(self, ctx: Any):
+        self.ctx = ctx
+        self._plugin_id = getattr(ctx, "plugin_id", "unknown")
+
+    def get_input_schema(self) -> Dict[str, Any]:
+        """默认从类属性 input_schema 取."""
+        schema = getattr(self, "input_schema", None)
+        return schema or {}
+
+    def collect_entries(self) -> Dict[str, EventHandler]:
+        """
+        默认实现：扫描自身方法，把带入口标记的都收集起来。
```

Ruff (0.14.8)

Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)